src/server/lib/example.js

```diff
@@ -6,3 +6,21 @@ exports.pluralize = function (count, singular, plural) {
   return `${count} ${plural}`;
 };
 
+
+exports.pluralize_v2 = function (count, singular, plural) {
+  if (count === 1) {
+    return `${count} ${singular}`;
+  }
+  plural = plural || `${singular}s`;
+  return `${count} ${plural}`;
+};
+
+exports.method = function (count) {
+  for(let i = 0; i < count; i++){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected space(s) after "for". (keyword-spacing)](https://app.codacy.com/app/Codacy-ACME/nodeproject/pullRequest?prid=4310063)